File: src/main/java/com/spring/dto/request/account/ChangePasswordRequest.java
package com.spring.dto.request.account;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class ChangePasswordRequest {
    @NotBlank
    @Size(min = 6, message = "Password must be at least 6 characters long")
    private String newPassword;

    @NotBlank
    @Size(min = 6, message = "Password must be at least 6 characters long")
    private String confirmPassword;
}

File: MSMA/app/src/main/java/vn/edu/usth/msma/data/dto/request/profile/ChangePasswordRequest.kt
package vn.edu.usth.msma.data.dto.request.profile

import com.google.gson.annotations.SerializedName

data class ChangePasswordRequest(
    @SerializedName("newPassword")
    var newPassword: String,

    @SerializedName("confirmPassword")
    var confirmPassword: String,
)